examples/sample/package.json

```diff
@@ -3,22 +3,28 @@
   "version": "1.0.0",
   "scripts": {
     "start": "expo start",
-    "android": "expo start --android",
-    "ios": "expo start --ios",
-    "web": "expo start --web"
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "web": "expo start --web",
+    "moti:sync": "cd ../.. && watchlist -e packages/moti -- sh -c 'cd examples/sample && sh moti-sync.sh'"
   },
   "dependencies": {
-    "expo": "~51.0.6",
-    "expo-status-bar": "~1.12.1",
-    "moti": "^0.27.2",
-    "react": "18.2.0",
-    "react-native": "^0.74.1",
-    "react-native-gesture-handler": "~2.16.1",
-    "react-native-reanimated": "~3.10.1"
+    "expo": "~54.0.0-preview.6",
+    "expo-dev-client": "~6.0.4",
+    "expo-linear-gradient": "~15.0.2",
+    "moti": "*",
+    "react": "19.1.0",
+    "react-native": "0.81.0",
+    "react-native-gesture-handler": "~2.28.0",
+    "react-native-reanimated": "~4.0.2",
+    "react-native-safe-area-context": "~5.6.0",
+    "react-native-worklets": "~0.4.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.0",
-    "babel-plugin-module-resolver": "^5.0.0"
+    "@aleclarson/watchlist": "0.3.5",
```

Comment:
Added my fork of `watchlist` here (pinned to specific version for security 😉 ), since I couldn't get it working with the original `watchlist` package. If you prefer a different tool, LMK.

This is used to watch a directory for changes, running a specified command when changes are detected. The `-e` flag means "run the command at the beginning too".